frontend/src/App.js

```diff
@@ -59,6 +61,26 @@ function App() {
   const [isAppLoading, setIsAppLoading] = useState(false);
   const [appLoadingText, setAppLoadingText] = useState('Loading');
   const [alert, setAlert] = useState(null);
+  const [notifications, setNotifications] = useState({ whatsNew: '' });
+
+  const [areThereUnreadNotifications, setAreThereUnreadNotifications] = useState((() => {
+    try {
+      const readNotifications = window.localStorage.getItem('whatsnew-read-notifications');
```

Comment:
Should we/can we move this to some sort of helper file?

Reply:
Probably a const 'whatsnew-read-notifications'

Reply:
Changed to a const. The function needs to be scoped inside the react component since it evaluates state.